Samples/ReVerC/Driver.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
+
+namespace Quantum.ReVerC
+{
+    class Driver
+    {
+        static void Main(string[] args)
+        {
+            using (var sim = new QuantumSimulator())
+            {
+                QArray<Result> a = new QArray<Result> { Result.One, Result.Zero };
```

Comment:
Would you be willing to add some comments here explaining how these two arrays represent the input into the `add` operation? Also, a quick C# hint, but you can use `var a` here to avoid repeating the type name `QArray<Result>` in the declaration and constructor call.